Development/xASL_test_BIDSFlavorsFull.m

```diff
@@ -66,19 +66,29 @@ function xASL_test_BIDSFlavorsFull(pathExploreASL,pathTest,bTest)
 	fprintf('Copying the source data to a temporary folder: %s\n',conversionPath);
 	if ~exist(conversionPath, 'dir')
 		mkdir(conversionPath);
-	end
-	
-	fList = xASL_adm_GetFileList(flavorsPath, [], 'List', [], 1);
+    end
+    
+    if xASL_exist(flavorsPath)
```

Comment:
You might add 'dir' to the exist function.

Reply:
Good idea :)